src/bitmessageqt/__init__.py

```diff
@@ -2480,46 +2858,85 @@ def click_actionSettings(self):
                 self.settingsDialogInstance.ui.lineEditNamecoinUser.text()))
             BMConfigParser().set('bitmessagesettings', 'namecoinrpcpassword', str(
                 self.settingsDialogInstance.ui.lineEditNamecoinPassword.text()))
-            
+
             # Demanded difficulty tab
             if float(self.settingsDialogInstance.ui.lineEditTotalDifficulty.text()) >= 1:
                 BMConfigParser().set('bitmessagesettings', 'defaultnoncetrialsperbyte', str(int(float(
-                    self.settingsDialogInstance.ui.lineEditTotalDifficulty.text()) * defaults.networkDefaultProofOfWorkNonceTrialsPerByte)))
+                    self.settingsDialogInstance.ui.lineEditTotalDifficulty.text()
+                ) * defaults.networkDefaultProofOfWorkNonceTrialsPerByte)))
             if float(self.settingsDialogInstance.ui.lineEditSmallMessageDifficulty.text()) >= 1:
                 BMConfigParser().set('bitmessagesettings', 'defaultpayloadlengthextrabytes', str(int(float(
-                    self.settingsDialogInstance.ui.lineEditSmallMessageDifficulty.text()) * defaults.networkDefaultPayloadLengthExtraBytes)))
-
-            if self.settingsDialogInstance.ui.comboBoxOpenCL.currentText().toUtf8() != BMConfigParser().safeGet("bitmessagesettings", "opencl"):
-                BMConfigParser().set('bitmessagesettings', 'opencl', str(self.settingsDialogInstance.ui.comboBoxOpenCL.currentText()))
+                    self.settingsDialogInstance.ui.lineEditSmallMessageDifficulty.text()
+                ) * defaults.networkDefaultPayloadLengthExtraBytes)))
+
+            if self.settingsDialogInstance.ui.comboBoxOpenCL.currentText().toUtf8(
+            ) != BMConfigParser().safeGet("bitmessagesettings", "opencl"):
+                BMConfigParser().set(
+                    'bitmessagesettings', 'opencl',
+                    str(self.settingsDialogInstance.ui.comboBoxOpenCL.currentText()))
                 queues.workerQueue.put(('resetPoW', ''))
 
             acceptableDifficultyChanged = False
-            
-            if float(self.settingsDialogInstance.ui.lineEditMaxAcceptableTotalDifficulty.text()) >= 1 or float(self.settingsDialogInstance.ui.lineEditMaxAcceptableTotalDifficulty.text()) == 0:
-                if BMConfigParser().get('bitmessagesettings','maxacceptablenoncetrialsperbyte') != str(int(float(
-                    self.settingsDialogInstance.ui.lineEditMaxAcceptableTotalDifficulty.text()) * defaults.networkDefaultProofOfWorkNonceTrialsPerByte)):
+
+            if any(
+                    [
+                        float(self.settingsDialogInstance.ui.lineEditMaxAcceptableTotalDifficulty.text()) >= 1,
```

Comment:
pep8 validation

Reply:
This is actually correct. All tools pass this and actually require four extra spaces here if you take them out. It's to do with the visual indentation compared to the contents of the conditional. Once the fab work is in you can run:
```
coffeedogs@tinternet:~/PyBitmessageSplit2$ fab -H localhost code_quality:fix=0,details=1,filename=src/bitmessageqt/__init__.py
[localhost] Executing task 'code_quality'          
                                     
total pycodestyle flake8 pylint path_to_file                               
                                                     
0 0 0 0 /home/coffeedogs/PyBitmessageSplit2/src/bitmessageqt/__init__.py               
pycodestyle:    
                                  
flake8:                                                    
                                                                                                    
pylint:                                                
                                                      
Disconnecting from localhost... done.
```

Reply:
May it be like
```python
if any([
    float(self.settingsDialogInstance.ui...
    ...
]):
```
?

Reply:
That causes a pylint warning:
```C:604, 0: Wrong hanging indentation before block (add 4 spaces).```
Personally I would be happy with the style you suggest. Disabling just this case without disabling all hanging indentation warnings would mean forking pylint though.